### PR TITLE
Added contributors/devel/help-wanted.md

### PR DESCRIPTION
**Pull request by alpha-san**

### CODE REVIEW

File: contributors/devel/help-wanted.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Help Wanted
+
+Items marked with the `help-wanted` label need to ensure that they are:
```
Comment (cblecker) [inline contributors/devel/help-wanted.md]: The label is actually `help wanted` now

Comment (cblecker) [reply contributors/devel/help-wanted.md]: Please also add a bit at the end about the `/help` and `/remove-help` commands (as detailed here: https://git.k8s.io/test-infra/commands.md)

Comment (alpha-san) [reply contributors/devel/help-wanted.md]: Okay cool. Updated help-wanted.md based on your feedback.

**Comment by alpha-san** [comment]
@cblecker squashed :)

**Pull request merged**